code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -142,16 +142,22 @@
 	if(shuttle.mode == SHUTTLE_CRASHED)
 		to_chat(user, SPAN_NOTICE("\The [src] is not responsive"))
 		return
-
+	var/remaining_time = timeleft(door_control_cooldown) / 10
+	to_chat(user, SPAN_WARNING("The shuttle is not responding, try again in [remaining_time] seconds."))
 	if(dropship_control_lost && skillcheck(user, SKILL_PILOT, SKILL_PILOT_EXPERT))
-		var/remaining_time = timeleft(door_control_cooldown) / 10
-		if(remaining_time > 60)
-			to_chat(user, SPAN_WARNING("The shuttle is not responding, try again in [remaining_time] seconds."))
-			return
```

Comment:
Shouldn't the mentions of Queens override be `Queen's override` (if we don't just call it something else)

Reply:
Have any suggestions?

Reply:
What do you think now?